adapters/roulax/roulax.go

```diff
@@ -0,0 +1,144 @@
+package roulax
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+
+type ExtImpRoulax struct {
+	Pid   string `json:"Pid,omitempty"`
+	PublisherPath string `json:"publisherPath,omitempty"`
+}
+
+
+
+// Builder builds a new instance of the Adot adapter for the given bidder with the given config.
```

Comment:
There seems to be a typo. replace `Adot` with `roulax`. Or better to remove this comment